### PR TITLE
Add Python39 support and drop Django2.1 support

### PR DESCRIPTION
**Pull request by hramezani**
## Problem

Explain the problem you are fixing (add the link to the related issue(s), if any).

## Solution

Explain the solution that has been implemented, and what has been changed.

## Commandments

- [ ] Write PEP8 compliant code.
- [ ] Cover it with tests.
- [ ] Update `CHANGES.rst` file to describe the changes, and quote according issue with `GH-<issue_number>`.
- [ ] Pay attention to backward compatibility, or if it breaks it, explain why.
- [ ] Update documentation (if relevant).

### CODE REVIEW

**Comment by hramezani** [comment]
Hi @jezdez, Could you please take a look at this PR and the other 4 PRs created by me Or ping somebody to take a look.
Thanks!

**Comment by hramezani** [comment]
@smithdc1 Do you have time to take a look at this?

File: tox.ini [hunk 1]
```diff
@@ -1,14 +1,13 @@
 [tox]
 envlist =
-    py{36,37,38}-django{21,22,30,31,trunk}
+    py{36,37,38,39}-django{21,22,30,31,trunk}
```
Comment (smithdc1) [inline tox.ini:3]: Drop 2.1.

File: setup.py [hunk 2]
```diff
@@ -45,8 +45,8 @@ def get_version(root_path):
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
         'Framework :: Django',
-        'Framework :: Django :: 2.1',
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
         'Framework :: Django :: 3.1',
```
Comment (smithdc1) [inline setup.py:52]: A couple of rows down is `test_require` this needs updating as well.

**Comment by hramezani** [comment]
Thanks @smithdc1. I updated the PR and addressed your comments.

**Review by smithdc1 — APPROVED** [review]

**Pull request merged**